validations/kyc.test.ts

```diff
@@ -58,6 +64,52 @@ describe('/kyc', () => {
       Object.values(KycStatus),
     )
   })
+  if (config.clientApiKey && config.providerId) {
+    it('sends kyc webhooks', async () => {
+      const beforeKycAdd = new Date().toISOString()
+      const wallet = ethers.Wallet.createRandom()
+      const fiatConnectClient = new FiatConnectClient(
+        {
+          baseUrl: config.baseUrl,
+          network: Network.Alfajores,
+          accountAddress: wallet.address,
+          apiKey: config.clientApiKey,
+        },
+        (message: string) => wallet.signMessage(message),
+      )
+      const loginResult = await fiatConnectClient.login()
+      expect(loginResult.isOk).to.be.ok
+
+      const addKycResult = await fiatConnectClient.addKyc(mockKYCInfo)
+      expect(addKycResult.isOk).to.be.true
+
+      const getKycResult = await fiatConnectClient.getKycStatus({
+        kycSchema: mockKYCInfo.kycSchemaName,
+      })
+      expect(getKycResult.isOk).to.be.true
+      expect(getKycResult.unwrap().kycStatus).to.be.oneOf(
+        Object.values(KycStatus),
+      )
+      await new Promise((resolve) => setTimeout(resolve, 3000)) // Wait a bit for API requests to process
+      const client = axios.create({
+        baseURL: VALORA_BASE_URL,
+        validateStatus: () => true,
+      })
+      const response = await client.get(
+        `${VALORA_WEBHOOK_HISTORY_ENDPOINT}/${config.providerId}`,
+      )
+      expect(response.status).to.equal(
+        200,
+        `Error fetching webhook history: ${JSON.stringify(response.data)}`,
+      )
+      const coorespondingWebhook = response.data.kycHistory.find(
+        (kyc: any) =>
+          kyc.kyc_status === getKycResult.unwrap().kycStatus &&
+          kyc.event_timestamp > beforeKycAdd,
```

Comment:
this seems sketchy. wouldn't it be safer to look for the wallet address? since that should always be different between different times this test is run